package.json

```diff
@@ -23,7 +23,9 @@
 		"@wordpress/format-library": "1.26.5",
 		"@wordpress/rich-text": "3.24.5",
 		"@wordpress/wxr": "file:packages/wxr",
+		"cheerio": "1.0.0-rc.5",
 		"file-saver": "2.0.5",
+		"moment": "2.29.1",
```

Comment:
A late note - we should probably stay away from momentjs, as it's super bloated. Some good alternatives listed [here](https://github.com/you-dont-need/You-Dont-Need-Momentjs).